### PR TITLE
Writing native bears tutorials: Parameters changed

### LINKED ISSUE #5109: Writing native bears tutorial: use the `format` string method in an example
In the first example code of the tutorial "Guide to Writing a Native Bear" (in the section "A Hello World Bear"), the call to `debug` should be passed one argument: a string that would be formatted using the methode `format`.

### PR DESCRIPTION
**Pull request by pkcod**
Closes #5109

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by arpit7714** [comment]
do commit 
git commit --amend
and write message 
 filename:<space>do changes

correct the changes

Closes https://github.com/coala/coala/issues/5109
and read the new comer guide of writing commit messages  http://api.coala.io/en/latest/Developers/Newcomers_Guide.html

**Comment by RJ722** [comment]
It's just been 4 days, relax @newbazz! 😛

File: coala [hunk 1]
```diff
@@ -1,28 +1,601 @@
...
-from coalib import assert_supported_version
-
-assert_supported_version()
-
-from coalib.coala import main
-# Stop ignoring
-
-if __name__ == '__main__':
-    sys.exit(main())
+Guide to Writing a Native Bear
```
Comment (jayvdb) [inline coala]: why are you changing this file called `coala`?

**Comment by jayvdb** [comment]
Moved to https://github.com/coala/coala/pull/5278 , but stale there also.

**Pull request closed without merging**